src/extension/features/toolkit-reports/pages/net-worth/component.jsx

```diff
@@ -30,22 +33,39 @@ export class NetWorthComponent extends React.Component {
 
   render() {
     return (
-      <div className="tk-flex tk-flex-column tk-flex-grow">
-        <div className="tk-flex tk-justify-content-end">
-          {this.state.hoveredData && (
-            <Legend
-              assets={this.state.hoveredData.assets}
-              debts={this.state.hoveredData.debts}
-              debtRatio={this.state.hoveredData.debtRatio}
-              netWorth={this.state.hoveredData.netWorth}
+      <div className="tk-flex-grow tk-flex tk-flex-column">
+        <div className="tk-flex tk-pd-05 tk-border-b">
+          <div>
+            <LabeledCheckbox
+              id="someId"
+              checked={this.state.inverseDebt}
+              label="Flip Debt"
+              onChange={this.toggleDebtDirection}
             />
-          )}
+          </div>
+        </div>
+        <div className="tk-flex tk-flex-column tk-flex-grow">
+          <div className="tk-flex tk-justify-content-end">
+            {this.state.hoveredData && (
+              <Legend
+                assets={this.state.hoveredData.assets}
+                debts={this.state.hoveredData.debts}
+                debtRatio={this.state.hoveredData.debtRatio}
+                netWorth={this.state.hoveredData.netWorth}
+              />
+            )}
+          </div>
+          <div className="tk-highcharts-report-container" id="tk-net-worth-chart" />
         </div>
-        <div className="tk-highcharts-report-container" id="tk-net-worth-chart" />
       </div>
     );
   }
 
+  toggleDebtDirection = () => {
+    this.setState(prevState => ({ inverseDebt: !prevState.inverseDebt }));
```

Comment:
would you mind storing this in local storage using our local storage helpers so that it's preserved across refresh and while flipping through reports?

Reply:
Facepalm moment, I removed the forked repo... 🤦‍♂️ I can try to restore the repo and make the changes shortly, or you can go in and make the edits.

Reply:
Well, the lovely support guys restored the repo. I'll add these changes soon. 😄

Reply:
@joshmadewell updates made into my latest commit. 😊